lib/controllers/meals_controller.js

```diff
@@ -21,22 +21,20 @@ const getResource = (request, response, next) => {
 }
 
 const postFoodResource = (request, response, next) => {
-    const meal_id = request.params.meal_id
-    const food_id = request.params.food_id
+    const { meal_id, food_id } = request.params
```

Comment:
i like this assignment of variables

be sure to write your code following camel case though since we're in JS land 👍